src/classes/BDI_DataImportFLSService_TEST.cls

```diff
@@ -184,6 +184,48 @@ private class BDI_DataImportFLSService_TEST {
                         'considered valid by the mocked BDI_DataImportFLSService.');
     }
 
+    /**
+     * @description This test validates that the CampaignMember.CampaignId gets
+     * exempted from Field Level Security checks to allow for campaign member imports
+     */
+    @IsTest
+    static void testCampaignIdIsValidateUpdateException (){
+        Set<BDI_DataImportFLSService.AccessLevel> accessLevels = new
```

Comment:
I think we still need the call to `UTIL_CustomSettingsFacade.enableAdvancedMapping();`, otherwise the call to `BDI_DataImportService.getDefaultFieldMapping()` on line 221 returns an instance of the help text mapping class.

Reply:
Oh, I just added it back